Remove proposals gem from Gemfile - not needed

davefrey opened the pull request:
The `decidim-assemblies` gem was already a dependency in the Gemfile.lock, unsure adding it to the Gemfile changes anything but it does make it explicit, and explicitly the Decidim version we want.

davefrey committed:
Add decidim-assemblies gem

oliverbarnes reviewed (changes requested):
Thanks, this made me dig a little to find out what exactly the decidim gem installs, and it looks like both assemblies and proposals are included. Here's an example Gemfile lock for the Sabadell instance (done by Codegram): https://github.com/AjuntamentDeSabadell/decidim-sabadell/blob/master/Gemfile.lock#L129-L148

And their Gemfile only declare decidim itself: https://github.com/AjuntamentDeSabadell/decidim-sabadell/blob/master/Gemfile#L7

I agree we should be explicit about which parts of decidim we're integrating with, but it's better to do it in docs. As for the version, all the dependencies are semantically versioned and follow decidim's own version.

So instead of adding the assemblies gem, how about we actually remove the proposals gem from here, and leave only the decidim and liquidvoting gems?

davefrey commented:
I agree, thanks for chasing that up, will push change in moment. Finally I think I like that approach, all the usual Decidim things are in by default, and we LV are clearly an opt-in extension.

davefrey committed:
Back off to just top-level Decidim gem

The dependencies we want, like proposals and assemblies, will be
loaded thanks to the decidim.gemspec dependencies.

davefrey commented:
And this reminds me that all those dependencies are included in the lock file because `gem "decidim"` in the `Gemfile` loads all the dependencies in the `decidim.gemspec` file.

Commit:
Merge branch 'master' into enable-assemblies

# Conflicts:
#	Gemfile
